lib/features/home/home.dart

```diff
@@ -240,12 +244,14 @@ class _HomePageState extends State<HomePage> with TrayListener, WindowListener {
             }),
         MenuItem.separator(),
         MenuItem(
-            key: 'exit',
-            label: 'exit'.i18n,
-            onClick: (item) {
-              windowManager.destroy();
-              LanternFFI.exit();
-            }),
+          key: 'exit',
+          label: 'exit'.i18n,
+          onClick: (item) async {
+            LanternFFI.exit();
+            await trayManager.destroy();
+            await windowManager.destroy();
```

Comment:
Updated the sequence that we use to clean up and destroy resources when exiting the app